tools/anthosbm-ansible-module/roles/bmctl-tool/tasks/main.yml

```diff
@@ -13,50 +13,42 @@
 # limitations under the License.
 
 ---
-- set_fact: bmctl_path="/usr/local/sbin"
```

Comment:
Th bmctl_path change was done by a previous PR to add flexibility to use different versions of bmctl tool. Please keep it if it did not cause any issues.

Reply:
I'm not sure what your review / feedback is here. What change are you referring to, or what do you think we could improve?